activemq-stomp/src/main/java/org/apache/activemq/transport/stomp/ProtocolConverter.java

```diff
@@ -280,11 +294,7 @@ protected void handleException(Throwable exception, StompFrame command) throws I
         // Let the stomp client know about any protocol errors.
         ByteArrayOutputStream baos = new ByteArrayOutputStream();
         PrintWriter stream = new PrintWriter(new OutputStreamWriter(baos, "UTF-8"));
-        if (exception instanceof SecurityException || exception.getCause() instanceof SecurityException) {
-            stream.write(exception.getLocalizedMessage());
-        } else {
-            exception.printStackTrace(stream);
-        }
+        exception.printStackTrace(stream);
```

Comment:
Why not keeping the original code here writing SecurityException to the stream ?

Reply:
Corrected